deformable_manipulation_experiment_params/launch/cloth_single_pole_params.launch

```diff
@@ -40,10 +43,11 @@
     <param name="cloth_num_control_points_y"    value="$(arg cloth_num_control_points_y)"   type="int"/>
 
     <!-- Cloth position params -->
-    <arg name="cloth_leading_edge_x"    value="0.06"/>
+    <arg name="cloth_leading_edge_x"    default="0.06"/>
+    <arg name="cloth_leading_edge_y"    default="0.00"/>
 
     <param name="cloth_com_x"           value="$(eval cloth_leading_edge_x + (cloth_x_size / 2.0))" type="double"/>
-    <param name="cloth_com_y"           value="0.0"                                                 type="double"/>
+    <param name="cloth_com_y"           value="$(eval cloth_leading_edge_y + 0.0)"                  type="double"/>
```

Comment:
To be discussed: would you prefer we leave this change here or not? It will make it easier to try some other params for the setup (don't need to compile every time). I have set the values of these new variables so that the setup is the same as that on matser.

Reply:
It's not about being able to change this here, it's about math. The math relationship between a (square, non-folded) cloth's COM and edge is `y_com + y_size/2 = y_boundary`. What you have written is `y_com = y_boundary`, thus `y_size = 0` which makes no sense. There should be a better way of specifying these parameters. For example, just put a raw number here if the COM is what you want to specify instead of the leading edge. For my use I wanted the leading edge to not move if I resized the cloth. For your use it seems that you want to keep the COM in the same spot if you resize the cloth.